### PR TITLE
#79 allow `=~` as OP_MATCHES operator in where clause conditionals

### PR DESCRIPTION
**Pull request by abynenkov-ib**

### CODE REVIEW

**Comment by rchowinfoblox** [comment]
Alexey you're gonna hate this, but I realized I recalled the wrong token for regexp operator in perl: it's `=~` (and `!~`), not `~=`.
If it's possible I think we should change to `=~` as that's what most people are used to.  And should we also support `!~` for negated regexp matching?

**Comment by wplu** [comment]
yes, please change it - i have changed the desc in the issue

**Comment by wplu** [comment]
we'll support !~ in a different issue

**Comment by abynenkov-ib** [comment]
Done, but `~=` seems better to me
And I agree with Phil, `!~` (not match) is not needed, because it's possible to use `not` operator in seal

File: docs/source/examples/petstore/petstore.all.seal [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# matches
+deny subject group regexp to use petstore.* where subject.iss =~ ".*regexp.*";
```
Comment (wplu) [inline docs/source/examples/petstore/petstore.all.seal:2]:
```suggestion
deny subject group everyone to use petstore.* where where subject.jti =~ "@petstore.swagger.io$";
```

File: docs/source/examples/petstore/petstore.all.test.rego [hunk 2]
```diff
@@ -1,5 +1,34 @@
 package petstore.all
 
+#deny subject group regexp to use petstore.* where subject.iss ~= ".*regexp.*";
+test_regexp {
+	in := {
+		"type": "petstore.pet",
+		"verb": "use",
+		"jwt": sealtest_jwt_encode_sign({
+			"iss": "just test regexp params",
```
Comment (wplu) [inline docs/source/examples/petstore/petstore.all.test.rego:9]: please use `jti` as in example from petstore.all.seal

File: docs/source/examples/petstore/petstore.all.test.rego [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+
+	deny with input as in
+}
+
+test_regexp_negative {
+	in := {
+		"type": "petstore.pet",
+		"verb": "use",
+		"jwt": sealtest_jwt_encode_sign({
+			"iss": "iss should not match",
```
Comment (wplu) [inline docs/source/examples/petstore/petstore.all.test.rego:23]: please use jti as in example from petstore.all.seal

**Review by wplu — CHANGES_REQUESTED** [review]
please change example in petstore.all.seal - otherwise LGTM!

File: docs/source/examples/petstore/petstore.all.seal (hunk 1 above)
Comment (abynenkov-ib) [reply docs/source/examples/petstore/petstore.all.seal]: done

**Review by wplu — APPROVED** [review]

**Pull request merged**